### PR TITLE
3:Changed refreshControl background to fix transparency issue on firs…

### PR DESCRIPTION
**Pull request by EMG114**
Changed refreshControl background to fix transparency issue on first cell. Timestamp label is not showing through the cell when you do pull refresh.

### CODE REVIEW

File: ParkenDD/Colors.swift [hunk 1]
```diff
@@ -65,6 +65,7 @@ extension UIColor {
 					red   = CGFloat((hexValue & 0xF00) >> 8)       / 15.0
 					green = CGFloat((hexValue & 0x0F0) >> 4)       / 15.0
 					blue  = CGFloat(hexValue & 0x00F)              / 15.0
+                    alpha = CGFloat(hexValue & 0x000F)             / 15.0
```
Comment (olivaresf) [inline ParkenDD/Colors.swift]: Make sure your indentation is the same as the project. I believe it's set to 4/4 spaces. The code you submitted may look good on Xcode but it's going to be distracting when reviewing it.

**Review by olivaresf — CHANGES_REQUESTED** [review]
Looks good other than the indentation!

File: ParkenDD/Colors.swift (hunk 1 above)
Comment (EMG114) [reply ParkenDD/Colors.swift:73]: Fixed indentation and submitted a commit with the changes.

**Review by EMG114 — COMMENTED** [review]
Fixed indentation and submitted a commit with the changes.

File: ParkenDD/Colors.swift [hunk 2]
```diff
@@ -9,86 +9,86 @@
...
+        if rgba.hasPrefix("#") {
+            let index   = rgba.characters.index(rgba.startIndex, offsetBy: 1)
+            let hex     = rgba.substring(from: index)
+            let scanner = Scanner(string: hex)
+            var hexValue: CUnsignedLongLong = 0
+            if scanner.scanHexInt64(&hexValue) {
+                switch hex.characters.count {
+                case 3:
+                    red   = CGFloat((hexValue & 0xF00) >> 8)       / 15.0
+                    green = CGFloat((hexValue & 0x0F0) >> 4)       / 15.0
```
Comment (EMG114) [inline ParkenDD/Colors.swift]: Indentation should be good now

**Comment by olivaresf** [comment]
Hmm, there are even more indentation changes now. Can you make sure Github doesn't show any changes?

**Review by EMG114 — COMMENTED** [review]
I had  to fixed Preferences in Xcode for indentation of tabs vs spaces

**Review by EMG114 — COMMENTED** [review]
Preferences was on space and not n tabs

**Review by EMG114 — COMMENTED** [review]
Tab is correct now. Finally after trying to figure out the issue with indentation, it was a tab issue.

**Comment by EMG114** [comment]
Finally fixed indentation

**Comment by EMG114** [comment]
Issue can be closed now.

File: ParkenDD/Colors.swift [hunk 3]
```diff
@@ -65,7 +65,7 @@ extension UIColor {
 					red   = CGFloat((hexValue & 0xF00) >> 8)       / 15.0
 					green = CGFloat((hexValue & 0x0F0) >> 4)       / 15.0
 					blue  = CGFloat(hexValue & 0x00F)              / 15.0
-                    alpha = CGFloat(hexValue & 0x000F)             / 15.0
+					alpha = CGFloat(hexValue & 0x000F)             / 15.0
```
Comment (EMG114) [inline ParkenDD/Colors.swift:68]: Fixed Indentation

**Review by EMG114 — COMMENTED** [review]
Tab is correct now on Xcode Preferences.

**Review by olivaresf — APPROVED** [review]

**Pull request merged**